File: MPI/mpicheck/tests/functions/1.0/MPI_Comm_split.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_comm_split(var_0, var_1, var_2, var_3, var_4)
       call pmpi_comm_split(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Comm_split.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    int var_2;
    MPI_Comm *var_3;
    int ret;
    /* calls */
    ret = MPI_Comm_split(var_0, var_1, var_2, var_3);
    ret = PMPI_Comm_split(var_0, var_1, var_2, var_3);
    return 0;
}
